bndtools.core/src/bndtools/editor/project/RunRequirementsPart.java

```diff
@@ -474,9 +512,7 @@ protected boolean performSelectionDrop(ISelection data, Object target, int locat
             }
 
             if (!adding.isEmpty()) {
```

Comment:
Here you leave the empty check before calling the new method. Above you remove the empty check. I would assume these should be consistent.